discovery-infra/tests/test_e2e_install.py

```diff
@@ -11,19 +14,28 @@ class TestInstall(BaseTest):
 
     @JunitTestSuite()
     @pytest.mark.parametrize("openshift_version", get_available_openshift_versions())
-    def test_install(self, get_nodes, get_cluster, openshift_version):
-        new_cluster = get_cluster(cluster_config=ClusterConfig(openshift_version=openshift_version), nodes=get_nodes())
+    def test_install(self, configs: Tuple[ClusterConfig, TerraformConfig], get_nodes, get_cluster, openshift_version):
```

Comment:
what will happened when we will add one more config? i am not sure i like this idea with 2 configs provided to test.
In my opinion idea of fixture is not to get function but to to get ready object.  I really don't see any reason to use here get_nodes or get_cluster as fixture in case you provide configs as fixture.

Reply:
You can always create them manually 
```suggestion
    cluster_config, tf_config = ClusterConfig(), TerraformConfig()
```